src/sidebar/util/groups.js

```diff
@@ -1,4 +1,25 @@
 import escapeStringRegexp from 'escape-string-regexp';
+import serviceConfig from '../service-config';
+
+/**
```

Comment:
I appreciate the thoroughness of this comment! But I wonder if it could be a little more concise, e.g.

```
Should users be able to leave private groups of which they
are a member? Users may leave private groups unless
explicitly disallowed in the service configuration of the
`settings` object.
```

Reply:
Ya, thats better :)